<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsForm</class>
 <widget class="QWidget" name="SerialSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Connection retries: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="connectionRetries">
          <property name="maximum">
           <number>999999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Times</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="scanAutostart">
        <property name="text">
         <string>Start scanning directly after connection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="errorWarning">
        <property name="text">
         <string>Show warning on Sensor Error</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkSupportedModes">
        <property name="text">
         <string>Check Supported Modes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoReconnect">
        <property name="text">
         <string>Auto reconnect on Sensor disconnection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection Raw Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="debugActiveCheck">
        <property name="text">
         <string>Activate Raw data Recording</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="sebugSettings" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="debugFolder"/>
            </item>
            <item>
             <widget class="QToolButton" name="browseButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Send Prefix</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sendPrefix"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Recieve Prefix</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="receivePrefix"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>debugActiveCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>sebugSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
